src/coreComponents/fileIO/coupling/ChomboCoupler.cpp

```diff
@@ -94,36 +109,79 @@ void ChomboCoupler::write( double dt )
 void ChomboCoupler::read( bool usePressures )
 {
   GEOSX_LOG_RANK_0( "Waiting for file existence: " << m_inputPath );
-  waitForFileExistence( m_comm, m_inputPath.data());
+  waitForFileExistence( m_comm, m_inputPath.data() );
 
   GEOSX_LOG_RANK_0( "File found: " << m_inputPath );
 
   if( usePressures )
   {
     GEOSX_LOG_RANK_0( "Reading pressures..." );
 
-    FaceManager * faces = m_mesh.getFaceManager();
+    FaceManager * const faces = m_mesh.getFaceManager();
+    NodeManager * const nodes = m_mesh.getNodeManager();
+
     const localIndex n_faces = faces->size();
-    const localIndex n_nodes = m_mesh.getNodeManager()->size();
+    const localIndex n_nodes = nodes->size();
 
     /* Build the face FieldMap. */
     FieldMap_out face_fields;
     real64 * pressure_ptr = faces->getReference< real64_array >( "ChomboPressure" ).data();
     face_fields["Pressure"] = std::make_tuple( H5T_NATIVE_DOUBLE, 1, pressure_ptr );
 
     FieldMap_out node_fields;
+    node_fields["position"] = std::make_tuple( H5T_NATIVE_DOUBLE, 3, m_referencePositionCopy.data() );
```

Comment:
It seems that when `usePressures == false` neither pressure nor position is being read, and this function does... nothing (other than removing the file)? Just making sure if that is the intent.

Reply:
Yeah that is the intent.